lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -177,8 +182,9 @@ class ConvertAtenMaxPool2dOp : public OpConversionPattern<AtenMaxPool2dOp> {
     Value maxPool2d, paddedInput;
     if (failed(createPoolingOp<linalg::PoolingNchwMaxOp>(
             op, rewriter, self, /*supportNonFPInput=*/false, ceilMode,
-            kernelSizeIntValues, strideInts, paddingInts, dilationInts,
-            smallestFPValueAttr, outTensorShape, paddedInput, maxPool2d)))
+            /*dimension*/2, kernelSizeIntValues, strideInts, paddingInts,
```

Comment:
nit: comment is missing an `=`: `/*dimension=*/`

Reply:
> 

Fixed